repos/jsutils/src/array/areShallowEqual.js

```diff
@@ -0,0 +1,27 @@
+/** @module array */
+
+import { validate } from '../validation'
+import { isArr } from './isArr'
+
+/**
+ * Checks if two arrays contain the same elements in the same order
+ * Shallowly-compared.
+ * @function
+ * @param {Array<*>} arr 
+ * @param {Array<*>} otherArr 
+ * @return {boolean} true if arrays are equal in order and elements. False otherwise, null if input are not both arrays
+ */
+export const areShallowEqual = (arr, otherArr) => {
+  const [ valid ] = validate({ arr, otherArr }, { $default: isArr }) 
+  if (!valid) return null
+
+  if (arr === otherArr) return true
+  if (arr.length !== otherArr.length) return false
+
+  for (let i = 0; i < arr.length; i++) {
```

Comment:
Instead of re-writing this logic, you can just import `shallowEqual` from `../collection/shallowEqual`

But really, this method seems kind of point less. It's just implementing the `shallowEqual` logic in another place.

Why don't you just import `shallowEqual` ?

Reply:
good point. I forgot we had that function! I will remove `areShallowEqual`.